client/common/sass/components/_article-carousel.sass

```diff
@@ -0,0 +1,20 @@
+.article-carousel
+	margin: 0
+	padding: 0
+	list-style-type: none
+	grid-auto-flow: row
+	+grid-container
+
+.article-carousel-item
+	grid-column: span 4
+
+// The "Latest Incidents" carousel collapses to multiple rows
+// over smaller viewport sizes
+.article-carousel-item--latest
+	grid-column: 1 / -1 // span the entire grid
+
+	+tablet-up
+		grid-column: span 4 // ½ of grid
+
+	+desktop-up
+		grid-column: span 3 // ¼ of grid
```

Comment:
This implementation is very close to [what I did with blog featured articles](https://github.com/freedomofpress/pressfreedomtracker.us/blob/redesign/client/common/sass/components/_blog-list.sass#L17-L62) using grid. I fell back on flex for the scrolling part, but since it's not part of this PR, I think this implementation aligns with my implementation. Once the scrolling part is implemented, I think we can safely swap up the classes from markup and everything should 90% work perfectly.